src/main/resources/static/js/custom.js

```diff
@@ -0,0 +1,20 @@
+$(document).ready(function(){
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["$" is not defined.](https://www.codacy.com/app/matikz1110/PetFinder/file/2642351516/issues/source?bid=3239464&fileBranchId=3321823#l1)